doc/en/components/general-changelog-dv-wc.md

```diff
@@ -10,6 +10,11 @@ namespace: Infragistics.Controls
 
 All notable changes for each version of {ProductName} are documented on this page.
 
+### **{PackageVerChanges-24-2-NOV}**
```

Comment:
I think there's a separate section bellow for native WebComponents bellow `{PackageCommon}`